public/package.json

```diff
@@ -3,21 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.10",
```

Comment:
Do you need to be removing these?

Reply:
i never touched this, could be from when i merged bambi's changes into mine?

Reply:
these changes will just go back and forth if you could make it so there are no changes in your commit (assuming they aren't needed) then it will be helpful

Reply:
I looked into it and this was replaced with a later version, that's why it was taken off. you can see the newer one lower down.